tests/cli/deployment/test_deployment_run.py

```diff
@@ -81,7 +110,45 @@ async def test_start_in_option_schedules_flow_run_in_future(
     assert scheduled_time == expected_start_time
 
 
-def test_displays_scheduled_start_time(deployment_name: str, monkeypatch):
+@pytest.mark.parametrize(
+    "start_in, expected_display",
+    [
+        ("10 minutes", "in 10 minutes"),
+        ("5 days", "in 5 days"),
+        ("3 seconds", "in a few seconds"),
+        (None, "now"),
+        ("1 year and 3 months", "in 1 year"),
+        ("2 weeks & 1 day", "in 2 weeks"),
+        ("27 hours + 4 mins", "in 1 day"),
+    ],
+)
+async def test_start_in_displays_scheduled_start_time(
+    deployment_name: str,
+    orion_client: prefect.OrionClient,
+    start_in: str,
+    expected_display: str,
+):
+
+    await run_sync_in_worker_thread(
+        invoke_and_assert,
+        command=[
+            "deployment",
+            "run",
+            deployment_name,
+            "--start-in",
+            start_in,
+        ],
+        expected_output_contains=["Scheduled start time:", expected_display],
+    )
+
```

Comment:
lns 144-148 are probably beyond the scope of this test

Reply:
Yeah I think you could exclude those

Reply:
I'll leave them in favor of [your other comment](https://github.com/PrefectHQ/prefect/pull/7928#discussion_r1052522906)